sdk/quantum/quantum-jobs/test/utils/testAuthentication.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { DefaultAzureCredential } from "@azure/identity";
+import { QuantumJobClient } from "../../src";
+import { env, record, RecorderEnvironmentSetup } from "@azure/test-utils-recorder";
+import TestClient from "./testClient";
+import { replaceStorageSig } from "./recorderUtils";
+
+export async function authenticate(that: any): Promise<any> {
+  const recorderEnvSetup: RecorderEnvironmentSetup = {
+    replaceableVariables: {
+      SUBSCRIPTION_ID: "677fc922-91d0-4bf6-0000-4274d319a0fa",
+      RESOURCE_GROUP: "resourcegroup",
+      WORKSPACE_NAME: "workspace",
+      AZURE_CLIENT_ID: "ce7bd34e-0000-0000-0000-000000000000",
```

Comment:
`AZURE_TENANT_ID` is missing in this list and is showing up in the recordings.
Please add an entry here for `AZURE_TENANT_ID` with a fake value and regenerate the recordings.

Reply:
I tried that but then I failed on the playback mode in two ways:
1) the tenant id that I put to replace was not passing a validation check done by the azure-identity library (it was throwing an exception)
2) the playback couldn't find a matching record for the login request
Do you have a suggestion to address the items above?
I was told the TenantID is not a super secret thing since with just it a malicious user cannot do anything (and it already shows up in other recordings, samples and documentation across many repos ).

Reply:
Thanks, I'll follow up on that to see what the problem is.

Reply:
Fixed.